packages/models/test/fixtures/persist-config.mock.ts

```diff
@@ -0,0 +1,8 @@
+import { PersistConfig } from '../../src';
+
+export function persistConfig(opt?: Partial<PersistConfig>): PersistConfig {
+  return {
+    outputDir: 'tmp',
+    ...(opt || {}),
```

Comment:
Object spread automatically omits non-objects.

```suggestion
    ...opt,
```